libsolidity/interface/StandardCompiler.cpp

```diff
@@ -70,6 +74,36 @@ Json::Value formatFatalError(string const& _type, string const& _message)
 	return output;
 }
 
+Json::Value formatSourceLocation(SourceLocation const* location)
+{
+	Json::Value sourceLocation;
+	if (location && location->source && !location->source->name().empty())
+	{
+		sourceLocation["file"] = location->source->name();
+		sourceLocation["start"] = location->start;
+		sourceLocation["end"] = location->end;
+	}
+
+	return sourceLocation;
+}
+
+Json::Value formatSecondarySourceLocation(SecondarySourceLocation const* _secondaryLocation)
+{
+	Json::Value secondarySourceLocation;
```

Comment:
Maybe here it makes more sense to do `if(!_secondLocation) return Json::Value();`